### ISSUE TITLE
Bug: XAML Controls Gallery (Debug): Accessibility: [MAS4.3.1]: In HC mode, Selected option is not visible when selected , in Expander and Pipspager cards

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4274]
Tags
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.3.1, HighContrast, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

**Steps to reproduce the bug**

1.Launch the XAML Controls Gallery (Debug) App
2.Go to the What's New-->Expander/PipsPager cards and activate.
3.Now navigate to Expand Direction/Orientation/Previous button visibility/next button visibility combo box with tab
4.Observe selected options are not visible when focused.

**Actual behavior**
Selected option is not visible when selected , in Expander and Pipspager cards

**Expected behavior**
 Everything should visible properly in HC mode.

**User Impact**
People with Low vision are not able to identify the selected option when the combo box is focused.

Version Info
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug)
![HCB 1](https://user-images.githubusercontent.com/74585893/108674502-5fca6600-74b3-11eb-9b9c-2f14177be5a3.png)
![HCB 2](https://user-images.githubusercontent.com/74585893/108674508-61942980-74b3-11eb-82d5-9d15849149e4.png)
![HCB 3](https://user-images.githubusercontent.com/74585893/108674511-635ded00-74b3-11eb-8e7b-c8c8bf37df76.png)
![HCB 4](https://user-images.githubusercontent.com/74585893/108674517-65c04700-74b3-11eb-989c-18468a4a8bd4.png)
![HCW 1](https://user-images.githubusercontent.com/74585893/108674584-87213300-74b3-11eb-87f3-5cacb63f9f6f.png)
![HCW 2](https://user-images.githubusercontent.com/74585893/108674585-88526000-74b3-11eb-862c-e96a201295fa.png)
![HCW 3](https://user-images.githubusercontent.com/74585893/108674587-88eaf680-74b3-11eb-8d9f-7b58efe19161.png)

**Comment by StephenLPeters** [comment]
@stmoy FYI
